### ISSUE TITLE
How to use this 

### ISSUE THREAD

**Issue by mohanen** [amacgregor/dot-files#3]
I have always been the IDE guy, never preferred vim. After looking you blog i thought i would give it go.
But don't know what to do with the cloned files.
should i link them to home or what

**Comment by astral451** [comment]
I also am attempting to use this, however in the clone and .zip download I noticed the .vim files under Bundle package are all 'folders.' and they won't load. I'm wondering if there is a need for the downloading and installation of the plugins directly.  Anyway it seems really promising, just struggling to get it to work.  FYI I'm a big NOOB when it comes to VIM and GIT so double+hard.

**Comment by mohanen** [comment]
It's uses vundle for package management. You just need bundle to install all the plugins . Look into vundle for more info

**Comment by JohnRandom** [comment]
Yeah, this isn’t so obvious at the first glance. All the plugins in the `vim/bundle/` directory are empty. This is because they are all git repositories themselves and as such, git doesn’t commit its contents. 

You also have to symlink the contents of this repo to your dotfiles. For the VIM stuff to work, you have to at least link
```.vimrc -> <path/to/repo>/vimrc```
and
```.vim -> <path/to/repo>/vim```
Then, you have to 
```git clone https://github.com/VundleVim/Vundle.vim.git ~/.vim/bundle/Vundle.vim```
yourself. This being done, you can open Vim and `:PluginInstall`, to automatically install all the plugins in `.vimrc`.

This made it work for me, but it isn’t straight forward in any fashion for new comers. For more information look into https://github.com/VundleVim/Vundle.vim.

**Comment by astral451** [comment]
I found a tid-bit on the vundle installation here as well. 
http://www.gmarik.info/blog/2011/chicken-or-egg-dilemma/

I used 
vim -u _vimrc +BundleInstall +qall

Since I'm on Windows I had to make a "_vimrc" file and a folder "vimfiles"  Took some work, but I think it's installing now.
